site_scons/prereq_tools/base.py

```diff
@@ -530,7 +530,7 @@ def run_build(self, opts):
         # argobots is not really needed by client but it's difficult to separate
         common_reqs = ['argobots', 'ucx', 'ofi', 'hwloc', 'mercury', 'boost', 'uuid',
                        'crypto', 'protobufc', 'lz4', 'isal', 'isal_crypto']
-        client_reqs = ['fuse', 'json-c']
+        client_reqs = ['fuse', 'json-c', 'capstone']
```

Comment:
I've just tested this and can confirm it's working as expected.

Reply:
Actually, if you need to repush then can you change https://github.com/daos-stack/daos/blob/master/src/client/dfuse/SConscript#L74 to be:
`    pil4dfsenv.AppendUnique(LIBS=['pthread', 'daos', 'dfs', 'duns', 'dl'])
      pil4dfsenv.require('capstone')
`

Reply:
ok. I will change it. Thank you!